core/grants.go

```diff
@@ -90,6 +97,7 @@ func (a *API) createGrant(ctx context.Context, x apiGrant) (*apiGrant, error) {
 		GuardData: data,
 		Policy:    g.Policy,
 		CreatedAt: g.CreatedAt,
+		Protected: g.Protected,
```

Comment:
Nah, in fact I think it's important to force that field to be false in the RPC, so that it's impossible for the client to create a protected grant.

If they made one, they'd be unable to revoke it. The only way protected grants are a reasonable thing to have is if they're completely managed by cored (both creating and deleting).